uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/shared/controllers/sub-controllers/leave-request.controller.js

```diff
@@ -5,23 +5,48 @@ define([
 ], function (controllers) {
   controllers.controller('LeaveRequestCtrl', LeaveRequestCtrl);
```

Comment:
it should be "Controller", as per the style guide

Reply:
Autofixed.